web/server.js

```diff
@@ -25,7 +15,8 @@ app.prepare().then(() => {
     //if we want to manually handle any routes or add middleware, do it here
 
     require('./utils/databasemigrations')(pool); //Will automatically look for new SQL scripts and execute them in order
-    server.get('*', handleNextRequest); 
+    server.get('*', handleNextRequest);
+    server.post('*', handleNextRequest); 
```

Comment:
I think we could also do `server.all('*', (req, res) => { return handleNextRequest(req, res) })`

Reply:
Yes you're right, good catch.